src/Utils/TimingUtils/useCountdown.js

```diff
@@ -0,0 +1,48 @@
+import { useState, useEffect } from 'react';
+import moment from 'moment';
+import 'moment-duration-format';
+
+function useCountdown(startTime, endTime) {
+  const [duration, setDuration] = useState(
+    moment.duration(endTime - startTime)
+  );
+  const [hasReachedZero , setHasReachedZero ] = useState(false);
+  useEffect(() => {
+    const interval = 1000;
+
+    let intervalDuration;
+    if (!hasReachedZero) {
+      intervalDuration = setInterval(() => {
+        setDuration(moment.duration(duration.asMilliseconds() - interval));
+      }, interval);
+    }
+
+    if (!hasReachedZero && duration.asMilliseconds() <= 0) {
+      setDuration(moment.duration(0));
+      setHasReachedZero(true);
+      clearInterval(intervalDuration);
+    }
+    return () => {
+      clearInterval(intervalDuration);
+    };
+  }, [duration, hasReachedZero]);
+
+  return duration.asMilliseconds();
+}
+
+function formmatCountdown(countdown){
```

Comment:
sorry for the misunderstanding, but please move to a new `hhmmssFormat.js` in the same directory as `useCountdown`.
the idea was that it might be useful for other parts of the project, so we'll make it easy to find.

(sorry to be that guy, but notice it's `format` and not `formmat`)